options/locale/locale_en-US.ini

```diff
@@ -3075,6 +3075,17 @@ users.list_status_filter.is_2fa_enabled = 2FA Enabled
 users.list_status_filter.not_2fa_enabled = 2FA Disabled
 users.details = User Details
 
+ips = Signup IPs
```

Comment:
`ips` seems an ambiguous word.

* `IPS` screen?
* Last login `IPs`?
* In-flight request `IPs`?

Reply:
```suggestion
```